app/controllers/users_controller.rb

```diff
@@ -8,9 +8,8 @@ def create
     user = User.new(user_params)
     if user.save
       session[:user_id] = user.id
+      UserActivatorMailer.welcome_email(user).deliver_now
```

Comment:
I think this would be another item for instructor clarification -- it works but I want to know why I wrote down the .with(user ...) syntax

Reply:
(Mostly a note to myself)